AppController/djinn.rb

```diff
@@ -2834,6 +2834,17 @@ def get_all_appengine_nodes()
     return ae_nodes
   end
 
+  # Gets a list of autoscaled nodes by going through the nodes array
+  # and splitting the array from index greater than the 
+  # minimum images specified.
+  def get_autoscaled_nodes()
+    autoscaled_nodes = []
+    min_images = Integer(@options['min_images'])
+    @state_change_lock.synchronize {
+      autoscaled_nodes = @nodes.drop(min_images)
```

Comment:
Should this function have the check for 'appengine' only nodes instead of scale down? Just in case nodes could get reordered?